README.md

```diff
@@ -34,9 +34,10 @@ Then install lld from an [LLVM nightly package](http://apt.llvm.org/)
 #### Windows
 Install [MSYS2](http://www.msys2.org/)
 
-In MSYS2 Shell:
+In mingw64 Shell:
 
-    pacman -S make git gcc bison flex llvm clang
+    pacman -S make git mingw-w64-x86_64-gcc bison flex mingw-w64-x86_64-llvm \
+    mingw-w64-x86_64-clang mingw-w64-x86_64-lld
```

Comment:
nit: This feels a bit like a workaround, rather than a fix.

Did you consult the MSYS people about this and asked why this would be necessary?
What did we conclude when we discussed this on Discord when Chris had the problems you are addressing?

The reason I'm asking, is that this now restricts us to 64 bit machines (although going to 32 bit is still easy). Whereas the MSYS line should have worked on 32 and 64 bit (and also worked on other pacman based environments).

*You might also want to move bison and flex to the left, so the unreadable part starts later.*